config.py

```diff
@@ -9,16 +9,26 @@ class Config(object):
     USE_SESSION_FOR_NEXT = True
     RESPONSE_OPERATIONS_UI_SECRET = os.getenv('RESPONSE_OPERATIONS_UI_SECRET', "secret")
     BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL')
+    SESSION_TYPE = "redis"
+    PERMANENT_SESSION_LIFETIME = os.getenv('PERMANENT_SESSION_LIFETIME', 43200)
+    REDIS_HOST = os.getenv('REDIS_HOST')
+    REDIS_PORT = os.getenv('REDIS_PORT')
+    REDIS_DB = os.getenv('REDIS_DB', 3)
```

Comment:
I think we should assume there is a Redis instance for backstage sessions, or for the response-operations-ui service rather than for all of RAS and default this to 0. If the app is using a shared Redis instance, this parameter can be set in the environment.

Reply:
Agreed, I'll change and make the service name configurable in the cloud_foundry.py file